modules/twinklewarn.js

```diff
@@ -1139,6 +1143,13 @@ Twinkle.warn.callback.change_category = function twinklewarnCallbackChangeCatego
 	if (value === 'singlenotice' || value === 'singlewarn') {
 		// no categories, just create the options right away
 		createEntries(Twinkle.warn.messages[value], sub_group, true);
+	} else if (value === 'singlecombined') {
+		var unSortedSinglets = $.extend({}, Twinkle.warn.messages.singlenotice, Twinkle.warn.messages.singlewarn);
```

Comment:
>should probably be written as Twinkle.warn.messages.singlenotice.

@siddharthvp Pinging you here rather than at https://github.com/azatoth/twinkle/pull/773#discussion_r352202964 since I'm resolving it here, but ✅ .

In fact, I just redid this section.  I initially waffled on whether this should be sorted, and only started figuring out sorting (and dealing with `messageData`) after `createEntries`, but just never went back and combined them, which should read clearer.